core/src/test/scala/org/apache/spark/DistributedSuite.scala

```diff
@@ -160,10 +160,6 @@ class DistributedSuite extends SparkFunSuite with Matchers with LocalSparkContex
     val data = sc.parallelize(1 to 1000, 10)
     val cachedData = data.persist(storageLevel)
     assert(cachedData.count === 1000)
-    assert(sc.getExecutorStorageStatus.map(_.rddBlocksById(cachedData.id).size).sum ===
```

Comment:
You could replace these with code based on `sc.statusStore`.

Reply:
Using sc.statusStore here would also cause the bug I mentioned above (rddStorageInfo.numCachedPartitions difference). This is why I left this part untouched.